### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kschmit90**
If the language is only optimized for the developer, the end user may suffer. Why not work towards an ideal? The language is the developers way of communicating with the computer, so it must be evenly optimized.

### CODE REVIEW

**Comment by scottefein** [comment]
I think we've moved to a point where our languages can both be performant and optimized for the developer. A lot of modern languages already meet this requirement. Go, Scala, etc.

**Comment by kschmit90** [comment]
Yes. Does that not make it germane to developer happiness?

**Comment by scottefein** [comment]
Languages have many use cases. A slower language might be fine depending on what I'm doing. The one thing I care about always would be that it's optimized for the developer. Ruby is a great example-Super hard to parse for the computer, but many use cases where it works great.

**Comment by kschmit90** [comment]
Ah, so you would like the language to always be optimized for the developer, relative to the use case. Makes sense.

**Comment by nathany** [comment]
NOT seems too strong to me.

How about "My language must be optimized for me rather than the computer."? Or before/over the computer? As you say, we can have both.
